packages/esm-patient-attachments-app/src/camera-media-uploader/camera-media-uploader-types.ts

```diff
@@ -17,3 +17,7 @@ export interface CameraMediaUploaderContextType {
   setError?: React.Dispatch<React.SetStateAction<Error>>;
   allowedExtensions?: Array<string> | undefined;
 }
+
+export interface FileAttachment extends UploadedFile {
+  capturedFromWebcam?: boolean;
+}
```

Comment:
I've added this property to `UploadedFile` to distinguish between file uploads and images uploaded via the webcam. Usually, I'd add this directly to Core instead of extended the interface in this way. I've elected to do this instead because of [this potentially problematic commit](https://github.com/openmrs/openmrs-esm-core/pull/1258).

Reply:
I've pushed a commit updating core so we can use the updated interface.